src/labels_transfer/knn/config.vsh.yaml

```diff
@@ -0,0 +1,103 @@
+functionality:
+  name: knn
+  namespace: "labels_transfer"
+  version: "dev"
+  description: "Performs label transfer from reference to query using KNN classifier"
+  authors:
+    - __merge__: ../../authors/vladimir_shitov.yaml
+      roles: [ author ]
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: "--input"
+          alternatives: ["-i"]
+          type: file
+          description: Input h5mu file
+          direction: input
+          required: true
+          info:
+            slots:
+              obsm:
+              - type: "double"
+                name: "X_scanvi"
```

Comment:
What is the first default that will be used? `X_pca`?

Reply:
What do you mean? In scvi-tools tutorials, they run classifiers on scVI/scANVI embeddings (same for HLCA). I guess, one could do it in PCA as well but need to make sure that the same PCA transformation was used for reference and query